src/assessment.js

```diff
@@ -49,5 +49,6 @@ var q2 = [/* your answer here */];
 // executing it.
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
+  var temp = deathstar.fire(); //unsure, but deathstar.fire(alderann) doesn't feel like a binding.
+  var blammo = temp.bind(alderann;)
```

Comment:
You're on the right track, but aren't quite there. When you use `bind`, the syntax is as follows.
```javascript
var boundFunc = funcToCall.bind(context);
boundFunc();
```